### PR TITLE
bayestestR: rnorm_perfect

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
@strengejacke @pdwaggoner 

very small and trivial blogpost mainly to test the absolute URL stuff.

### CODE REVIEW

**Review by strengejacke — COMMENTED** [review]
Looks good!

File: content/posts/bayestestR_rnorm_perfect.Rmd [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(knitr)
+options(knitr.kable.NA = '')
+```
+
+
+
+Many times, for instance when teaching, I needed to quickly and simply generate a perfectly **perfectly normally distributed sample** to illustrate or show some of its characteristics.
```
Comment (pdwaggoner) [inline content/posts/bayestestR_rnorm_perfect.Rmd]: The word "perfectly" is used twice here

File: content/posts/bayestestR_rnorm_perfect.Rmd [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+library(tidyverse)
+
+x %>% 
+  density() %>%  # Compute density function
+  as.data.frame() %>% 
+  ggplot(aes(x=x, y=y)) +
+  geom_line()
+```
+
+We can also easily color some of the parts of the curve, for instance, the observations lying beyond +2 standard deviation.
```
Comment (pdwaggoner) [inline content/posts/bayestestR_rnorm_perfect.Rmd]: "deviation" at the end of the sentence should be plural: "deviations"

**Comment by pdwaggoner** [comment]
Just reviewed it. Only two minor grammatical changes. Other than that, looks great to me!

File: content/posts/bayestestR_rnorm_perfect.Rmd (hunk 1 above)
Comment (strengejacke) [reply content/posts/bayestestR_rnorm_perfect.Rmd]: Maybe everybody should get that it's really _perfect_! :-D But I also stumbled upon this.

Comment (pdwaggoner) [reply content/posts/bayestestR_rnorm_perfect.Rmd]: How about: "perfectly, perfectly, perfectly"? If two is great, then three must be... perfect (?). That's sound logic right? ;)

**Comment by DominiqueMakowski** [comment]
Will merge tomorrow morning. I hope it will work 🤞

**Comment by DominiqueMakowski** [comment]
Merged, thanks your your comments guys :)

**Comment by DominiqueMakowski** [comment]
Based on last time it seems like r-bloggers updates in my evening, so expect to see the post there in a few hours

**Pull request merged**